src/Icon/__tests__/Icon.test.js

```diff
@@ -45,4 +46,26 @@ describe('<Icon />', () => {
       expect(tree).toMatchSnapshot();
     });
   });
+
+  it('sets arbor icon class names', () => {
+    const wrapper = mountWithTheme(<Icon name="test-icon-name" />);
+
+    expect(wrapper.find('StyledIcon')).toHaveClassName('ar ar-test-icon-name');
+  });
+
+  it('does not clobber arbor icon class names when passing css prop', () => {
+    const wrapper = mountWithTheme(
+      <Icon css={{ cursor: 'pointer' }} name="test-icon-name" />
+    );
```

Comment:
This is the relevant setup.  Without the fix, the `className` generated for the `css` prop ends up overriding the `className` in the component that sets the icon class names.  Now we peel it off of `props` and merge it into the other class names.